### DISCUSSION TITLE
Who is your favorite writer? 好きな作家を教えてください

### DISCUSSION THREAD

**Discussion by Y-eng** [NAIST-SE/AttractingContributors#491] [General]
Please tell me your favorite writer!

link: https://github.com/Y-eng/AttractingContributors-writer

**Comment by tmymzkr** [comment]
I done pull request 
please help me
https://github.com/m-tomoya/AttractingContributors-movie

**Reply by Y-eng** [reply]
Thanks for helping me!
I created a pull request to your repository.
